roles/variables/defaults/main.yml

```diff
@@ -164,7 +164,7 @@ config_base_path: "{{ (archive_config_base_path | regex_replace('\\/$','')) if i
 config_prefix: "/etc"
 
 ### Boolean to have cp-ansible download the Confluent CLI
-confluent_cli_download_enabled: "{{rbac_enabled}}"
+confluent_cli_download_enabled: true
```

Comment:
so as I understand, confluent_cli_download_enabled is independent of rbac_enabled only for CLI v3 ?

Reply:
we use cli only for secrets protection. Since this was dependant on rbac, we put this condition. Now, if secret protection is independent, I feel, the default value should be `secrets_protection_enabled`